config/Dockerfiles/tests.d/libvirt/02_libvirt-storage

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning for all supplied providers
+# distros.exclude: fedora26 centos7 fedora28
+# providers.include: libvirt
+# providers.exclude: none
+
+## NOTE: This is a temporary test to run libvirt tests until we fix
+## the networking race condition. Tests will only run on centos7.
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TEST_NAME="${DISTRO}/${PROVIDER}/02_libvirt-storage"
+DESCRIPTION="Test ${PROVIDER} provider with storage provisioning"
+
+echo "Test Name: ${TEST_NAME}"
+echo "${DESCRIPTION}"
+
+TARGET="libvirt-storage"
+
+pushd docs/source/examples/workspace
+
+function clean_up {
+    set +e
+    #linchpin -w . -p PinFile.${PROVIDER}.yml -v destroy ${TARGET}
+    # remove output file used to gen uhash if it exists
+    rm storage.txt || true
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+linchpin -w . -p PinFile.${PROVIDER}.yml -v up ${TARGET} 2> storage.txt
+
+cat storage.txt
+uhash=$(cat storage.txt | grep ${TARGET} | tail -n1 | cut -f3)
+name="centos71-$uhash"
+vm="${name}_0"
+
+for f in $(virsh dumpxml $vm | grep "source file"); do
+    if [[ $f = *${name}* ]]; then
+        # make sure the file exists
+        stat $(grep -o "'.*'" <<< $f | sed -e "s/'//g")
+    fi
+done
```

Comment:
Trying to decide the best course of action here. Is a loop necessary? You have the name of the system. It feels like a single test should be sufficient. 

Additionally, on my system, I have to pass `-c qemu:///system` because I have more than one qemu connection configured. Should we add that to the command that does `dumpxml`? It passes on the container, but may not let local testing occur without extra work.

Reply:
The loop is so that each disk can be `stat`ed.  As the tests and topologies are right now, there are three disks that are checked